internal/command/action_sequence_functions/global.lua

```diff
@@ -79,5 +79,13 @@ return {
         runner.extendTimelineSelection(move_function, args)
       end
     },
-  }
+  },
+  vkb = {
```

Comment:
using non shortened variable names improves readability, so I would rename this to virtual_keyboard_binding

Reply:
Agree!